widgetssdk/src/main/java/com/glia/widgets/GliaWidgets.java

```diff
@@ -54,8 +54,17 @@ public class GliaWidgets {
     /**
      * Use with {@link android.os.Bundle} to pass in the id of the queue you wish to enroll in
      * as a navigation argument when navigating to {@link com.glia.widgets.chat.ChatActivity}
+     *
+     * @deprecated Use {QUEUE_IDS} instead.
      */
+    @Deprecated
     public static final String QUEUE_ID = "queue_id";
+
+    /**
+     * Use with {@link android.os.Bundle} to pass in the ids of the queues you wish to enroll in
+     * as a navigation argument when navigating to {@link com.glia.widgets.chat.ChatActivity}
+     */
+    public static final String QUEUE_IDS = "queue_ids";
```

Comment:
NIT: I would add to javadocs a clarification on whether the queue IDs should be passed as an Array or List

Reply:
Actually, if I understood it right then it is a bit confusing since in other places it is a list and an array. Would it be reasonable to change it to list in the Intent parcele?

Reply:
It is always List.